static/esp_auto.css

```diff
@@ -255,6 +255,10 @@ table.multirow td {
 table.multirow tr:nth-child(odd) {
 	background-color:#e8e8e8 
 }
+table.sub {
+    max-width:100px;
```

Comment:
Max width of 100 px?

Reply:
jep... this is kind of a workaround for preventing the table to take the whole space and therefore destroying  the rest of the appearance. it could also be 0px or 100000px.
Actually it would maybe even better to make an "standalone" table out of it like the values table...